db/queries.py

```diff
@@ -1,23 +1,33 @@
-import init_django_orm  # noqa: F401
+from django.db.models import QuerySet, Q, F
 
-from django.db.models import QuerySet
+import init_django_orm  # noqa: F401
+from db.models import LoyaltyProgram, LoyaltyProgramParticipant, Customer
 
 
 def all_loyalty_program_names() -> QuerySet:
-    pass
+    return LoyaltyProgram.objects.all().values_list("name", "bonus_percentage")
 
 
 def not_active_customers() -> QuerySet:
-    pass
+    return LoyaltyProgramParticipant.objects.filter(
+        last_activity__gt="2021-01-01",
+        last_activity__lt="2022-01-01").values("customer__first_name")
```

Comment:
fix identation here

Reply:
to be honest, I don't understand what you mean? ) what exactly have I fix, flake doesn't show me any problem, and I dont understand what I have to do